elastic/kibana
# [Cloud Security] Added labels popover

## Summary

Added expand button for labels with popover:

https://github.com/user-attachments/assets/80950f51-b45b-4174-9be2-267b6aca569b

https://github.com/user-attachments/assets/690ef85b-be48-42df-bf00-02ee7d9303f2

**How to test**

To test this PR you can run

```
yarn storybook cloud_security_posture_packages
```

To test e2e

- Enable the feature flag

`kibana.dev.yml`:

```yml
uiSettings.overrides.securitySolution:enableVisualizationsInFlyout: true
xpack.securitySolution.enableExperimental: ['graphVisualizationInFlyoutEnabled']
```

- Load mocked data:

```
node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/logs_gcp_audit \ 
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601

node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/security_alerts \
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601
```

- Make sure you include data from Oct 13 2024. (in the video I use Last year)

### Checklist
